docs/site/Getting-started.md

```diff
@@ -42,13 +42,13 @@ Answer the prompts as follows:
 ? Project root directory: (getting-started)
 ? Application class name: StarterApplication
 ? Select features to enable in the project:
-❯◉ Enable tslint
- ◉ Enable prettier
- ◉ Enable mocha
- ◉ Enable loopbackBuild
- ◉ Enable vscode
- ◉ Enable repositories
- ◉ Enable services
+❯◉ Enable tslint: adds a linter with pre-configured lint rules
```

Comment:
I think it would be better to use the same tense in both parts of the "sentence", i.e.

```
Enable tslint: add a linter with pre-configured lint rules
```